### DISCUSSION TITLE
tmap8-opt: command not found

### DISCUSSION THREAD

**Discussion by truecruiseraurora** [idaholab/TMAP8#179] [Q&A] [answered]
Hello! I use Windows 11. I installed WSL, conda, MOOSE and TMAP8 (tmap7 "installation" was much easier...) using this guide (https://mooseframework.inl.gov/TMAP8/getting_started/installation.html). I successfully tested tmap8, running in WSL this commands:

make -j4
./run_tests -j4

And I have the problem stated in the title. When I open WSL, it looks like this:
![image](https://github.com/user-attachments/assets/329e1f8d-cfeb-419a-99b6-f46e5b93aab8)

I want to try to run a test file so I run this set of commands (I've copied ver-1a.i to the TMAP8 directory):

source ~/.zshrc
conda activate moose
cd ~/projects/TMAP8
tmap8-opt -i ver-1a.i

And after that WSL looks like this:

![image](https://github.com/user-attachments/assets/253430fb-b386-4d46-be1e-d7cf3194b3be)

Where did I go wrong? Any ideas?.. I would be very grateful for your help!

**Comment by simopier** [comment]
I'm surprised to see the `conda: command not found` On top of the terminal. especially if `conda activate moose` then runs. 
Did the `./run_tests -j4` command really lead to all the tests passing? If so, then the TMAP8 executable should be working.

What does 
```
cd ~/projects/TMAP8
ls
```
return?

@loganharbour @cticenhour, do you know what could have caused this issue on WSL?

**Reply by truecruiseraurora** [reply]
Command `ls` returnes these names of files:

![image](https://github.com/user-attachments/assets/c1314320-afcc-4f76-80b3-8d377b2abd26)

When I run `./run_tests -j4`, I get 49 passed tests, 23 skipped, 0 pending and 0 failed

![image](https://github.com/user-attachments/assets/f49b6845-4aac-4115-9d98-c5d58b379ea2)

**Reply by simopier** [reply]
I'm very puzzled because you indeed do not have `tmap-opt` under ~/projects/TMAP8, which tells me that TMAP8 might not have been fully installed. But the tests run fine... which I'm not sure how that happens without tmap8-opt. 
@cticenhour @loganharbour, any idea?

**Reply by ke7kto** [reply]
Maybe there's a separate directory things are getting found in... What happens when you run `make METHOD=opt` from the TMAP8 directory (with conda's moose environment active)?

**Reply by simopier** [reply]
@truecruiseraurora, I just wanted to follow up on this, did you run @ke7kto's instructions? 

We could also jump on a call to directly help you debug this issue.

**Reply by truecruiseraurora** [reply, marked as answer]
Sorry, I reinstalled windows, install tmap8 againg and now it works, so I don't need these instructions anymore...

**Reply by simopier** [reply]
Good to hear the issue was fixed once windows was re-installed!

Let us know if you have any questions as you use TMAP8.
